conf/minion

```diff
@@ -310,7 +310,7 @@
 # recon_randomize: True
 #
 # Each minion will have a randomized reconnect value between 'recon_default'
-# and 'recon_default + recon_max', which in this example means between 1000ms
+# and 'recon_default + recon_max', which in this example means between 1000ms6
```

Comment:
was this change a typo?

Reply:
That'll be me failing to type in the search box